tests/test_invessel_build.py

```diff
@@ -104,14 +104,19 @@ def test_ivb_basics(invessel_build):
 def test_ivb_construction(invessel_build):
 
     num_components_exp = 2
+    len_loci_pt_exp = 3
 
     remove_files()
 
     invessel_build.populate_surfaces()
     invessel_build.calculate_loci()
     invessel_build.generate_components()
 
+    rib_loci = invessel_build.Surfaces['component'].get_loci()[0]
```

Comment:
should we also test `ivb.get_loci()`?

Reply:
`ivb.get_loci()` just loops over the surfaces and uses the `Surface.get_loci()` method, I figured that it would be ok to test just at one surface